src/js/view/utils/skin.js

```diff
@@ -1,6 +1,11 @@
 import { prefix } from 'utils/strings';
 import { css, getRgba } from 'utils/css';
 
+export function getSkinName(url) {
+    // Regex locates the characters after the last slash, until it encounters the file extension.
+    return url.replace(/^(.*\/)?(.*)-?.*\.(css)$/, '$2');
```

Comment:
```suggestion
    return url.replace(/^(.*\/)?(.*)-?.*\.(css).*$/, '$2');
```

Nice! I suggest also removing all characters after `.css` (ie, url parameters, etc).